src/storage/db_storage.hpp

```diff
@@ -33,6 +33,12 @@ enum StatusDbField : uint8_t {
   DbMinorVersion
 };
 
+enum PbftMgrPreviousRoundStatus {
```

Comment:
"enum PbftMgrPreviousRoundStatus :  uint8_t" is  preferred

Reply:
fixed, thanks